crates/bevy_render/src/render_resource/storage_buffer.rs

```diff
@@ -101,6 +104,16 @@ impl<T: ShaderType + WriteInto> StorageBuffer<T> {
         self.label.as_deref()
     }
 
+    /// Set the buffer usage of the buffer.
+    ///
+    /// This method only allows addition of flags to the default usage flags.
+    ///
+    /// The default values for buffer usage are `BufferUsages::COPY_DST` and `BufferUsages::STORAGE`.
+    pub fn set_usage(&mut self, usage: BufferUsages) {
```

Comment:
Now that we're documenting this as an additive API,  IMO it should be renamed to `add_usages` for better code clarity.